step2-04/demo/src/index.spec.tsx

```diff
@@ -1,11 +1,7 @@
 import React from 'react';
 import { mount } from 'enzyme';
-import { TestMe } from './TestMe';
 
-describe('TestMe Component', () => {
-  it('should have a non-clickable component when the origina InnerMe is clicked', () => {
-    const wrapper = mount(<TestMe name="world" />);
-    wrapper.find('#innerMe').simulate('click');
-    expect(wrapper.find('#innerMe').text()).toBe('Clicked');
+describe('index', () => {
```

Comment:
Might want to add a more useful example test here (the file seemed to have been copy-pasted from the TestMe test)